### ISSUE TITLE
Macro crash with !say and multiple variables

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#94]
Ok... I don't have as much side information this time.
```
!roll 1d20 + @me.proficiency_bonus + @me.dexterity_mod + 2 >> ${atk1} = ${1}
!roll 1d20 + @me.proficiency_bonus + @me.dexterity_mod + 2 >> ${atk2} = ${2}
!roll 1d8 + @me.dexterity_mod >> ${dmg1} = ${3}
!target "Select target."
${tName} = @target.name |
${tAC} = @target.ac |
!say ${atk1} " | " ${atk2} " vs. " ${tName} "'s AC (" ${tAC} ")"
$atk1 > ${tAC} ? !say "Hit " ${tName} " for " ${dmg1} " points of slashing damage." :|
```
![image](https://user-images.githubusercontent.com/7004392/34850915-7417ebba-f6ed-11e7-81d9-ffcabc61dbf3.png)

The issue is specifically `!say ${atk1} " | " ${atk2} " vs. " ${tName} "'s AC (" ${tAC} ")"` - without that line, this macro does correctly output 
![image](https://user-images.githubusercontent.com/7004392/34850939-84a614c0-f6ed-11e7-9b7e-fb6ca72fd79c.png)

I also can make 
```
${atk1} = 20 |
${atk2} = 10 |
${tName} = "Test" |
${tAC} = 18 |
!say ${atk1} " | " ${atk2} " vs. " ${tName} "'s AC (" ${tAC} ")"
```
work, outputting
![image](https://user-images.githubusercontent.com/7004392/34850981-a38cb600-f6ed-11e7-9629-81cbba474998.png)

Thinking it might have been a missing double-quote or that it might be misinterpreting the single quote after `${tName}`, I tried replacing it with the simpler `!say ${atk1} " | " ${atk2}`, as well as just `!say ${atk1} ${atk2}`, to no avail - it still failed out.

**Comment by tlackemann** [comment]
Is this still a problem with interpolated strings?

**Comment by Pjb518** [comment]
I tested `!say "${atk1} | ${atk2}"` and everything worked as expected. I believe we're no longer supporting thing in the style of `!say ${atk1} " | " ${atk2}`, so I think this is safe to close.

**Comment by Aquaricat** [comment]
Confirming that `!say ${atk1} | ${atk2}` is working, holding this open pending the fixes in #106 & #107 since they're directly related to the expected functionality.

**Comment by Aquaricat** [comment]
Closing now that #106 & #107 have been cleared as well.
